module/VuFind/src/VuFind/Bootstrapper.php

```diff
@@ -396,6 +245,26 @@ protected function initTheme()
         $this->events->attach('dispatch', $callback, 9000);
     }
 
+    /**
+     * Initializes i18n services.
+     *
+     * @return void
+     */
+    protected function initI18n()
+    {
+        // Language not supported in CLI mode:
+        if (Console::isConsole()) {
+            return;
+        }
+
+        $this->events->attach(
+            'dispatch', function (MvcEvent $event) {
+                $event->getApplication()->getServiceManager()
+                    ->get(I18nInitializer::class)->init();
+            }, 5000
+        );
+    }
+
```

Comment:
This triggers the actual initialization. If additional language directories need to be considered these have to be added beforehand.